rllib/tests/test_supported_spaces.py

```diff
@@ -60,12 +61,43 @@
     ),
 }
 
+# The action spaces that we test RLModules with
+RLMODULE_SUPPORTED_ACTION_SPACES = ["discrete", "vector1d"]
+
+# The observation spaces that we test RLModules with
+RLMODULE_SUPPORTED_OBSERVATION_SPACES = ["discrete", "vector1d", "image"]
+
+DEFAULT_OBSERVATION_SPACE = DEFAULT_ACTION_SPACE = "discrete"
+
 
 def check_support(alg, config, train=True, check_bounds=False, tf2=False):
     config["log_level"] = "ERROR"
     config["env"] = RandomEnv
 
     def _do_check(alg, config, a_name, o_name):
+        if alg == "PPO":
```

Comment:
We decided to not have this algo == "PPO" specialization, right? 
We should check for which obs / action space to choose based on whether the enable_rl_module_api is set or not.